### DISCUSSION TITLE
A Short Hike

### DISCUSSION THREAD

**Discussion by SchizoSauce** [skyline-emu/skyline#1099] [Game Requests]
I don't know how to help, but if there's a way to support you guys financially, let me know. I have seen your efforts and it is quite fascinating to see and is actually quite impressive considering this is the only one that is legitimately favorable IMO, (unlike the OTHER certain emulator). I wish you guys the best and god speed!

And uhmmm also, I just wanna know if A short hike will be playable in the future, it's... it's a masterpiece. I know it's not easy but I just wanna let you guys know, it exists and it's fantastic 🥺.

**Comment by PixelyIon** [comment]
Thanks for being willing to support us, at the moment there isn't a way but there likely will be in the near future.
